showcase/src/app/demo/index.ts

```diff
@@ -2,3 +2,6 @@ export * from "./action-bar";
 export * from "./example-viewer";
 export * from "./table";
 export * from "./demo.module";
+export * from "./action-bar/action-bar.component";
+export * from "./button/button.component";
+export * from "./example-viewer/example-viewer.component";
```

Comment:
This re-export is not needed. This is already done in this barrel: `demo/example-viewer/index.ts`